javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,48 @@
+//Student manager
+const class07Students = [];
+function addStudentToClass(studentName)
+{
+    if(studentName === "")
+    {
+        console.log("please provide student name");
+    }
+    else if(studentName === "Queen Margrethe")
+    {
+        class07Students.push(studentName);
+        console.log(studentName + " has been added to class" )
+    }
+    else if(class07Students.length < 6)
```

Comment:
this only allows 5 people in the class, the max amount is 6.

Reply:
done now